diff --git a/src/qt/networkstyle.cpp b/src/qt/networkstyle.cpp
index 09d2df6958..4c4fa0ed32 100644
--- a/src/qt/networkstyle.cpp
+++ b/src/qt/networkstyle.cpp
@@ -20,6 +20,7 @@ static const struct {
     {ChainType::MAIN, QAPP_APP_NAME_DEFAULT, 0, 0},
     {ChainType::TESTNET, QAPP_APP_NAME_TESTNET, 70, 30},
     {ChainType::TESTNET4, QAPP_APP_NAME_TESTNET4, 70, 30},
+    {ChainType::TESTNET5, QAPP_APP_NAME_TESTNET5, 220, 20},
     {ChainType::SIGNET, QAPP_APP_NAME_SIGNET, 35, 15},
     {ChainType::REGTEST, QAPP_APP_NAME_REGTEST, 160, 30},
 };
